### PR TITLE
feat: [PROD-10458] add tooltip in scenario parameters

### PR DESCRIPTION
**Pull request by nborde-CSM**
Before creating a PR, please check that:
- [ ] Code is clean
- [ ] Tests are still working (cypress tests and `yarn test`)
- [ ] Changes don't cause new errors or warnings in browser console
- [ ] Documentation is up-to-date
- [ ] Breaking changes are clearly identified with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)

### CODE REVIEW

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

File: src/utils/TranslationUtils.js [hunk 1]
```diff
@@ -37,6 +41,12 @@ const addTranslationParametersLabels = (parameters) => {
       const label = parameter.labels[lang];
       resources[lang][key] = label;
     }
+    for (const lang in parameter?.options?.tooltipText) {
+      resources[lang] = resources[lang] || {};
+      const key = getParameterTooltipTranslationKey(parameter.id);
+      const label = parameter.options.tooltipText[lang];
+      resources[lang][key] = label;
```
Comment (esasova) [inline src/utils/TranslationUtils.js:48]: Maybe we can shorten this to lines in one: `resources[lang][key] = parameter.options.tooltipText[lang] `?
It was already like that in the previous loop but we could use the occasion to refactor it a bit :slightly_smiling_face:

**Review by esasova — APPROVED** [review]
just one remark, LGTM :+1:

File: src/utils/TranslationUtils.js (hunk 1 above)
Comment (nborde-CSM) [reply src/utils/TranslationUtils.js:48]: Indeed. I also make this change at line 41. :+1:
